### DISCUSSION TITLE
docker image

### DISCUSSION THREAD

**Discussion by Ho-Tung** [IvanaXu/TianChiProj#10] [Q&A] [answered]
> LDocker, https://tianchi.aliyun.com/competition/entrance/231759/introduction
can i get the docker image ?

**Comment by IvanaXu** [comment, marked as answer]
`docker pull ghcr.io/ivanaxu/tianchiproj/ldocker:1.0`

**Reply by Ho-Tung** [reply]
Dockerfile ?

**Reply by IvanaXu** [reply]
https://github.com/IvanaXu/TianChiProj/tree/master/LDocker
